variables.md

```diff
@@ -86,6 +86,8 @@ LIVECD | boolean | false | Indicates live image being used.
 LIVE_INSTALLATION | boolean | false | If set, boots the live media and starts the builtin NET installer.
 LIVE_UPGRADE | boolean | false | If set, boots the live media and starts the builtin NET installer in upgrade mode.
 LIVETEST | boolean | false | Indicates test of live system.
+LTP_RUN_NG_BRANCH | string | | Define the branch of the LTP_RUN_NG_REPO.
```

Comment:
```suggestion
LTP_RUN_NG_BRANCH | string |master | Define the branch of the LTP_RUN_NG_REPO.
```

Reply:
I avoided it as this is part of the module. Maybe overthinking but someone can use another module and the defaults might be different.